recipes/servicestack/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "servicestack" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4874987b82e81a8eeb146b53df6b282f48f843aa1e4f17eecc24d723dc3c5247
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - dataclasses >=0.6
+    - dataclasses-json >=0.5.4
+    - marshmallow >=3.12.2
+    - python >=3.9
+    - requests >=2.25.1
+
+test:
+  imports:
+    - servicestack
+    - tests
+  requires:
+    - dataclasses >=0.6
+    - dataclasses-json >=0.5.4
+    - pytest
+    - requests >=2.25.1
```

You don't need to duplicate any packages that are already listed under the `run` requirements.
```suggestion
  requires:
    - pytest
```
And unless you were wanting to have the recipe run the full unit test suite (which this isn't yet doing), I'd just remove the entire `requires` section.